main.tf

```diff
@@ -180,3 +180,9 @@ resource "aws_service_discovery_service" "metastore_proxy" {
     failure_threshold = 1
   }
 }
+
+resource "aws_route53_zone_association" "secondary" {
```

Comment:
I think we also need specify vpc_region
https://www.terraform.io/docs/providers/aws/r/route53_zone_association.html#vpc_region
current code may not work when deploying to non default regions.

Reply:
But we will always set default region when declaring AWS_PROFILE?
But its good to put the region to make sure code doesn't break. Updating PR